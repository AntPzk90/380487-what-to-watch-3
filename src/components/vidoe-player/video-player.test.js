import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './../vidoe-player/video-player.jsx';

const film = {
  title: `Shutter island`,
  src: `shutter-island.jpg`,
  poster: `shutter-island.jpg`,
  titlePoster: `shutter-island.jpg`,
  genre: `Drama`,
  releaseDate: `2019`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`SnapshotTest VideoPlayer`, () => {
  const tree = renderer
  .create(<VideoPlayer
    film = {film}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
