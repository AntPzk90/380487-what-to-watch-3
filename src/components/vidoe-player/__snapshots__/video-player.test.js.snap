// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest VideoPlayer 1`] = `
<video
  autoPlay="autoPlay"
  className="small-movie-card__image"
  muted="muted"
  poster="img/shutter-island.jpg"
/>
`;
